src/structurizr/model/component.py

```diff
@@ -82,7 +82,7 @@ class Component(StaticStructureElement):
     def __init__(
         self,
         *,
-        parent: "Container",
+        parent: "Container" = None,
```

Comment:
If you set a default of `None` please annotate the type as optional.

```suggestion
        parent: Optional["Container"] = None,
```